ow_plexil/src/plexil-adapter/PlexilPlanSelection.h

```diff
@@ -0,0 +1,34 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef Plexil_Planner_H
+#define Plexil_Planner_H
+
+#include <ros/ros.h>
+#include <ow_plexil/PlanSelectionCommand.h>
+#include "OwExecutive.h"
+#include "OwInterface.h"
+
+class PlexilPlanSelection{
+  public:
+    PlexilPlanSelection() = default;
+    PlexilPlanSelection(const PlexilPlanSelection&) = delete;
+    PlexilPlanSelection& operator = (const PlexilPlanSelection&) = delete;
```

Comment:
Good!  The only thing missing is the destructor, which could also be a 'default'.